### DISCUSSION TITLE
Variable expansion during config check

### DISCUSSION THREAD

**Discussion by effemmeffe** [Flexget/Flexget#3174] [Q&A]
If I run config check I get the following error:
`2021-10-04 13:54:38 CRITICAL manager                       [/tasks/move-series/move/to] `` does not exist`

The move section in my task move-movies is this:
```
    move:
      to: "{? folder.root ?}{? folder.movies ?}{{trakt_movie_name|default(movie_name)|pathscrub}} ({{trakt_movie_year|default(movie_year)}})"
      rename: "{{trakt_movie_name|default(movie_name)|pathscrub}} ({{trakt_movie_year|default(movie_year)}}){% if quality|default(False) %} - [{{quality}}]{% endif %}"
```

Obviously the path folder.root or folder.movies are not correct in my variable definition.
Is there some way I can see what is the actual path that the config sees?

**Comment by lamski** [comment]
Try running `flexget --test execute` - this should automatically set the logging level to verbose and do a test run using your configuration. You might be able to see the paths used.
